src/mpv_calendar/constants.py

```diff
@@ -25,7 +25,11 @@ class Constant:
     last_playing_file = CACHE / ".mpv_playing"
     mpv_socket = "/tmp/mpvsocket"  # nosec
     port = 8080
-    no_default_image_fallback: bool = True
+
+    # debug flags
+    default_image_fallback_instant = True
+    default_image_fallback_calendar = True
+    default_image_fallback_reset = False
```

Comment:
After some testing, these are probably the right settings, so they don't need to be gated as flags anymore, but this change was to facilitate the experiment.

There are 3 situations in which mpv is triggered to add an item to the playlist:
1. instant play / instant queue
2. calendar queue
3. resetting to the default image/video

When adding/queuing an item, the default item is added. There is a lua hook as well to fall back, but it doesn't always trigger. It's future work to figure out how to resolve the logic that these flags control and that hook.